provider-tutorials/provider-tutorial.md

```diff
@@ -62,7 +62,7 @@ After installing all required components you will be asked to set up your node.
 
 `Ethereum wallet address (default=internal wallet):` - Paste your own Ethereum address to which you have private keys stored. If you leave this space empty an address will be created for you on your local system.
 
-`price NGNT per hour (default=5):` - Type in the value of renting your computer power as a provider. You can use default price \(5 NGNT per hour\) by leaving this field empty. **This command shows up only when running GolemSP for the first time**
+`price tGLM per hour (default=5):` - Type in the value of renting your computer power as a provider. You can use default price \(5 tGLM per hour\) by leaving this field empty. **This command shows up only when running GolemSP for the first time**
```

Comment:
AFAIK this should be 'GLM' not 'tGLM' because providers will starting on mainnet by default.

Reply:
Well it is hard to say because it also mentions `rinkeby` in a few places:
- "For this release we have prepared a dedicated and controlled sub-network using Ethereum Rinkeby Testnet"
- "To confirm correctness of the listed values head over to [https://rinkeby.etherscan.io/](..."

Reply:
I think this is beyond our scope of responsibility but let's discuss it with whoever is responsible for this tutorial.

Reply:
it should be `GLM` and `default=0.1`
```suggestion
`price GLM per hour (default=0.1):` - Type in the value of renting your computer power as a provider. You can use default price \(0.1 GLM per hour\) by leaving this field empty. **This command shows up only when running GolemSP for the first time**
```